ansible/roles/load-balancer/templates/keepalived.conf.j2

```diff
@@ -0,0 +1,22 @@
+vrrp_instance VI_1 {
```

Comment:
Pls look at the keepalived.conf [here](http://docwiki.cisco.com/wiki/OpenStack_Havana_Release:_High-Availability_Manual_Deployment_Guide#Keepalived_.26_HAProxy)

I think it's a good idea to implement the haproxy tracking script implemented in the above guide. The script will force a vrrp switch over when the local haproxy process dies.